simtools/tests/test_visualize.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+
+import logging
+import os
+import sys
+import numpy as np
+from astropy import units as u
```

Comment:
I'm not using 'u' as namespace for units because I don't think single letter namespace is a smart thing to do. We can live well with 'units'.
I'm trying to keep single letters or double repeated letters variables for very short scopes, like short loops.

Reply:
Agreed. I took the use of a single letter u from ctapipe, but I don't mind using "units". I also don't use any other single/double letter variables, even in short loops.

Reply:
But this is standard and I disagree. Almost every code you find using units is using 'as u' (same is numpy is np). 

Using units already produces a lot of 'clutter' in the code. I would advocate to allow the single letter import here.

Reply:
I agree with using the standard ones for np, plt etc. But u really does not seem like a good idea to me.

Reply:
Now Gernot managed to convince me that u is actually a better idea in this case =)